cpu/nrf9160/Kconfig

```diff
@@ -37,4 +37,6 @@ config HAS_CPU_NRF9160
     help
         Indicates that the current cpu is 'nrf9160'.
 
+rsource "vectors/Kconfig"
+
```

Comment:
I think you are missing: `source "$(RIOTCPU)/nrf5x_common/Kconfig"`. As that one already includes the `cortexm_common` Kconfig you can remove it from here.

Reply:
@leandrolanzieri The problem here is that nRF9160 doesn't fully reuse nrf5x_common for now because some work is still needed on the driver side. That's why this CPU defines by itself which features it has instead of relying on nrf5x_common. 
If this is problem, I can have a look to fix the remaining issues with the shared drivers first, then let nRF9160 fully relies on nrf5x_common. What do you think ?

Reply:
But if we don't select `CPU_COMMON_NRF5X` its features should not be provided, right? This is currently failing because `MODULE_NRF5X_COMMON_PERIPH` is missing (defined in `cpu/nrf5x_common/periph/Kconfig.nrf5x`)